data/brands/24s.yaml

```diff
@@ -0,0 +1,17 @@
+name: 24S
+status: avoid
+description: |-
+  **24S** is owned by **LVMH**.
+
+  Bernard Arnault, the chairman and **CEO** of **LMVH** Moet Hennessy-Louis Vuitton, and the world's richest man according to Forbes, has **invested** in cybersecurity firm, **Wiz**, the **Israeli** maker of cloud security solutions for enterprises.
+reasons:
+- operations_in_israel
```

Comment:
Not going to comment on all of these, but here for example, I think there are plenty of people who don't find a need to boycott just because something is owned by LVMH whose CEO has invested in an Israeli company.

This is one reason why we have the `reasons` field.. maybe we should have a reason like `owner_invests_in_israeli_companies`?  Something that consumers of the dataset can filter out if they don't believe it's worthy of a boycott.

Ultimately I fear that this level of boycott might leave very little _not_ on the list.

Reply:
I completely agree, people will have different personal opinions on what is or isn't enough of a reason to boycott. As per your recent Discord message I think it is still worth including these types of brands as long as we can distinguish them from brands that are more "actively" pro-Israel

Reply:
Reasons don't 100% solve the different opinions problem. 

A board of directors of a company that invested $20K in a startup is very different from a board of directors of a company that invested $100 million. Or a company that is operating in the occupied land but neutral is very different from a company that is operating and taking a strong political stance. We need something that measures the intensity. What makes the cut should be manually curated 

If we take a step back, do we want to have a simpler and more obvious list to boycott or do we want to get all the information possible and all the customization?

I'd argue less is more. If you're not opinionated about what makes and what doesn't make the cut then you leave that option to the app developers which results in an ineffective boycott

![image](https://github.com/TechForPalestine/boycott-israeli-consumer-goods-dataset/assets/157234570/16a165f4-7a78-4896-8fbe-65f4dd959acb)

It seems to me like a prioritization problem, you want to find for every person what is the top brands to boycott so we need to have some brand stack ranking. If we collectively do it then we are able to cover the majority of the top brands. How do brands get to the top? That would be a manual effort to ensure quality and ease of boycott. If you don't solve this ranking problem here then you pass that problem to the next app developer to figure out which would result in different lists and ineffective boycotts, wdyt 🤔

Reply:
tbh i just want to merge this as a start.. we can do incremental improvements where ppl actually go thru the data and change the reasons for all LVMH brands for example.. i was thinking those could be follow up tasks

and to @tfp-anon1's point, i think it's fine if apps don't use every field but i like that the model can support more details and info.. app developers can decide how they use this data and display it

personally i don't want to rank.. i think of this as more like providing the data for a user to make an informed decision.. imo each can rank for themselves

Reply:
It's fine if you want to provide the data only and then there's an API layer that decides the ranking – we can spin this off within TFP – and then this data is a blackbox to outsiders with the goal being that the API layer on top of this is the one that decides the rankings without having another API exposing this data outside otherwise we end up with more division and boycotting becomes less effective, are you okay with that?

Reply:
Yh, up to you how you use the data and what you include in the API 👍

Thank you